Gemfile

```diff
@@ -11,6 +11,9 @@ gem 'parallel_tests'
 # Debugging
 gem 'pry'                                   # Easily debug from your console with `binding.pry`
 
+# Code style
+gem 'rubocop'
```

Comment:
@Fivell Thanks for having a look! I think we need it in both environments (Travis env is `test` by default, local env is `development` by default). That's why I included it globally.